src/Sylius/Bundle/AdminBundle/Controller/ImpersonateUserController.php

```diff
@@ -73,7 +75,13 @@ public function impersonateAction(Request $request, string $username): Response
 
         $this->addFlash($request, $username);
 
-        return new RedirectResponse($request->headers->get('referer'));
+        $redirectUrl = $request->headers->get('referer');
```

Comment:
I suppose we can do it with the default value as the second parameter:
```php
$redirectUrl = $request->headers->get(
    'referer',
    $this->router->generate('sylius_admin_customer_show', ['id' => $user->getId()])
);
```

Reply:
Thanks :)